### PR TITLE
how-to/upgrade: improve wording and format of upgrade docs

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Improve the wording and format of the upgrade document

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs/pull/1689

### Which version does your change affect? 
dev, v3.1, v3.0

### CODE REVIEW

File: dev/how-to/upgrade/from-previous-version.md [hunk 1]
```diff
@@ -185,29 +185,34 @@ ansible-playbook local_prepare.yml
 
 ## 滚动升级 TiDB 集群组件
 
-- 如果当前集群版本 < 3.0 并且 `process_supervision` 变量使用默认的 `systemd` 参数，则通过 `excessive_rolling_update.yml` 滚动升级 TiDB 集群。
+为优化 TiDB 集群组件的运维管理，从 TiDB 3.0.0 版本开始对 `systemd` 模式下的 `PD service` 名称进行了调整。
```
Comment (lilin90) [inline dev/how-to/upgrade/from-previous-version.md]: Do you think users need to know this info?

File: dev/how-to/upgrade/from-previous-version.md [hunk 2]
```diff
@@ -185,29 +185,34 @@ ansible-playbook local_prepare.yml
...
 
-    {{< copyable "shell-regular" >}}
+1. 如果 `process_supervision` 变量使用默认的 `systemd` 参数：
```
Comment (lilin90) [inline dev/how-to/upgrade/from-previous-version.md]: Not an ordered step.

```suggestion
- 如果 `process_supervision` 变量使用默认的 `systemd` 参数：
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 3]
```diff
@@ -185,29 +185,34 @@ ansible-playbook local_prepare.yml
...
+    - 当前集群版本 ≥ 3.0.0，滚动升级及日常滚动重启 TiDB 集群，使用 `rolling_update.yml`。
 
-- 如果 `process_supervision` 变量使用的是 `supervise` 参数，无论当前集群为哪个版本，均通过 `rolling_update.yml` 来滚动升级 TiDB 集群。
+        {{< copyable "shell-regular" >}}
+
+        ```bash
+        ansible-playbook rolling_update.yml
+        ```
+
+2. 如果 `process_supervision` 变量使用的是 `supervise` 参数，无论当前集群为哪个版本，均通过 `rolling_update.yml` 来滚动升级 TiDB 集群。
```
Comment (lilin90) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
- 如果 `process_supervision` 变量使用的是 `supervise` 参数，无论当前集群为哪个版本，均通过 `rolling_update.yml` 来滚动升级 TiDB 集群。
```

**Comment by juliezhang1112** [comment]
All comments addressed. PTAL again~ Thanks! @lilin90

**Pull request merged**